src/index.html

```diff
@@ -32,16 +31,29 @@ <h2>Recent Scores</h2>
           <div class="heading">
             <h2>Add your Score</h2>
           </div>
-          <form class="input-form">
-            <input type="text" placeholder="Your name" class="input" /> <br />
-            <input type="number" placeholder="Your Score" class="input" />
+          <form class="input-form form" id="score-form">
+            <input
+              type="text"
+              placeholder="Your name"
+              class="input name"
+              required
+            />
             <br />
+            <input
+              type="number"
+              placeholder="Your Score"
+              class="input score"
+              required
+            />
+            <br />
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a feature that cleans your `input` fields once the user has added a new score to make your App more professional.